akka-http-tests/src/test/scala/akka/http/scaladsl/server/TcpLeakApp.scala

```diff
@@ -35,12 +36,12 @@ object TcpLeakApp extends App {
         .toMat(Sink.head)(Keep.right).run())
     .last
     .onComplete {
-      case error ⇒
+      error ⇒
```

Comment:
I think it was just expected this demo app to fail, renaming to "result" makes sense IMO